cmd/boulder-va/main.go

```diff
@@ -101,6 +79,12 @@ func main() {
 	tlsConfig, err := c.VA.TLS.Load(scope)
 	cmd.FailOnError(err, "tlsConfig config")
 
+	if c.VA.DNSTimeout.Duration == 0 {
+		cmd.Fail("'dnsTimeout' is required")
+	}
```

Comment:
According to the docs a timeout of exactly `0` is a valid configuration:
> A Timeout of zero means no timeout.

Also, we should use `<= 0` to catch negative integers here as that would be an invalid configuration.

Reply:
This is a copy/paste duplicate line problem. This chunk exists on lines 57-59 anyways and should be removed here.

Reply:
I've changed the operator to `<=`.